src/webots/nodes/utils/WbNodeUtilities.cpp

```diff
@@ -194,6 +194,8 @@ namespace {
           return true;
         else if (WbNodeUtilities::hasARobotAncestor(node))
           return true;
+        else if (dynamic_cast<const WbGroup *>(node))
```

Comment:
Given that this block is executed only for `children` fields, is not possible to simplify and remove all the various checks instead of adding an additional condition that seems to be always true?